tests/test_derivatives.py

```diff
@@ -713,6 +713,16 @@ def test_deriv_spec(self):
         assert dxy0.x0 == {y: y+y.spacing/2}
         assert dxy02.x0 == {x: x+x.spacing/2}
 
+    def test_deriv_stagg_plain(self):
```

Comment:
Would it be worth also testing evaluating `staggered=None` onto `staggered=x`? Also potentially mixing `staggered=None` and `staggered=NODE`